charts/k8s-service/values.yaml

```diff
@@ -142,6 +142,22 @@ shutdownDelay: 5
 #         value: docker
 sideCarContainers: {}
 
+# canary specifies test pod(s) that are deployed alongside your application's stable track pods.
+# It is useful for testing a new release candidate in a production environment with minimal disruption and
+# for allowing you to find any issues early.
+#
```

Comment:
Since the schema for this var is known, this should include a table describing the full schema, like https://github.com/gruntwork-io/helm-kubernetes-services/blob/master/charts/k8s-service/values.yaml#L56

Reply:
Fixed in 25a016d